.github/workflows/eslint.yml

```diff
@@ -7,10 +7,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: hallee/eslint-action@1.0.3
-        # GITHUB_TOKEN in forked repositories is read-only
-        # https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-event-pull_request
-        if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository }} 
-        with:
-          repo-token: ${{secrets.GITHUB_TOKEN}}
-          source-root: web
+      - name: Install modules
+        run: yarn
+      - name: Register Matcher
+        run: echo "::add-matcher::.github/eslint-compact.json"
```

Comment:
I've never seen this before! What is the compact file doing?

Reply:
It's supposed to add annotations. I am having trouble getting it working and might simplify.

https://github.com/actions/toolkit/blob/main/docs/commands.md#problem-matchers